source/extensions/filters/http/cache/cacheability_utils.cc

```diff
@@ -0,0 +1,60 @@
+#include "extensions/filters/http/cache/cacheability_utils.h"
+
+#include "common/common/utility.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace Cache {
+
+namespace {
+
+bool static isCacheableStatusCode(absl::string_view status_code) {
```

Comment:
s/static// (you are already in an anon namespace.

Also see above comment about not making these kinds of functions private so that you can unit-test them.

Reply:
Will remove the static keyword.
This is also a function solely used and tested by another function (i.e isCacheableResponse). It was also @toddmgreer to move them to unnamed namespaces (which I agree with). What do you think Todd?

Reply:
I will replace the function with a CONSTRUCT_ON_USE getter of the hash set.